src/card.css

```diff
@@ -0,0 +1,90 @@
+.card {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  display: flex;
+  flex-direction: column;
+  
+  --font-weight-l: 700;
+  --font-weight-m: 500;
```

Comment:
use a separate file for variables if you use them outside of this block. (which you do)